src/main/java/com/conveyal/gtfs/loader/StringListField.java

```diff
@@ -34,7 +35,10 @@ public ValidateFieldResult<String> validateAndConvert(String original) {
     public Set<NewGTFSError> setParameter(PreparedStatement preparedStatement, int oneBasedIndex, String string) {
         // FIXME
```

Comment:
Out of scope, but can probably be removed.

Reply:
Yeah I agree, not even sure what the fix me was here. Removed it in https://github.com/conveyal/gtfs-lib/pull/386/commits/bc1e1b7051acc7a8c9c149c2453bdf38bb94a2f9

Reply:
My guess is it was there because of that nasty regex